lib/core/src/sdk.rs

```diff
@@ -830,13 +830,28 @@ impl LiquidSdk {
                     compressed: true,
                     inner: keypair.public_key(),
                 };
+                let webhook = self
+                    .persister
+                    .get_webhook_url()?
+                    .map(|url| SubmarineWebhook {
+                        url,
+                        hash_swap_id: Some(true),
+                        status: Some(vec![
+                            SubSwapStates::InvoiceFailedToPay,
+                            SubSwapStates::SwapExpired,
+                            SubSwapStates::TransactionClaimed,
+                            SubSwapStates::TransactionClaimPending,
```

Comment:
The pending is needed so the user app wakes up and cooperate with the swapper claim, that's understood.
But the TransactionClaimed will then invoke another webhook call, do we need it? I am concerned that in most cases the user will get an unnecessary notification.
In the case of non-cooperative claim by the swapper (which should be very rare) perhaps it is acceptable that when the user opens the app the refund will be broadcasted?

Reply:
Yeah, TransactionClaimed would normally be the final status so could be left until the app is opened. Handling of this status validates the preimage and sets the swap to complete.